### PR TITLE
Wrench Supermatter Shard to the ground

### PR DESCRIPTION
**Pull request by SpironoZeppeli**
I doubt this code is the standard method of doing things, but trying to understand this code is a bit AIDS

[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: SpironoZeppeli
add: You can now wrench the supermatter shard
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

The supermatter shard is nearly impossible to work with due to it moving about in changing atmospheres. As such, the ability to secure it to the ground is necessary.

### CODE REVIEW

**Comment by praisenarsie** [comment]
This is gonna be closed and you should know why yourself.

**Comment by SpironoZeppeli** [comment]
Thanks for being so informative!

**Comment by Armhulen** [comment]
@praisenarsie rude!

**Comment by Kromgar** [comment]
@SpironoZeppeli you know it's going to be closed because being able to drag the supermatter around will only result in death and chaos

**Comment by SpironoZeppeli** [comment]
It's only the shard that can be un/secured, not the crystal.

Updated PR, thanks to Naksu on discord for the help

**Comment by 81Denton** [comment]
>@SpironoZeppeli you know it's going to be closed because being able to drag the supermatter around will only result in death and chaos

The shard is the small thing on a pedestal that starts unwrenched by default

The crystal is the big immovable thinga and unaffected by this PR

**Comment by praisenarsie** [comment]
That still brings kromgar's point right back around.

**Comment by 81Denton** [comment]
What did he mean by this

**Comment by SpironoZeppeli** [comment]
@praisenarsie You know you can already move the shard around right?

**Comment by 81Denton** [comment]
Shard = Small thinga on a pedestal that comes in a cargo crate and is draggable by default

Crystal = Big immovable thinga that is unaffected by this PR

**Comment by ExcessiveUseOfCobblestone** [comment]
Does this work during testing?

**Review by Jalleo — COMMENTED** [review]
See above.

**Comment by Anonmare** [comment]
Dragging the crystal around honestly wouldn't be that big of an issue, there's actually only a slight obstacle with doing the same with an SM shard (That being the credits cost and breaking the crate open in cargo) and all it takes is one person on disarm intent to bump into you to dust you - assuming you'd get very far without a rad suit or in one for that matter.

File: code/modules/power/supermatter/supermatter.dm [hunk 1]
```diff
@@ -552,6 +554,10 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_shard)
 
 		radiation_pulse(src, 150, 4)
 
+/obj/machinery/power/supermatter_shard/wrench_act(mob/user, obj/item/tool)
```
[the remark this answers is not in the document]
Comment (ExcessiveUseOfCobblestone) [reply code/modules/power/supermatter/supermatter.dm]: He's differentiating with the movable var, although i'm not a fan of the approach.

Long term we would want to elevate the crystal to the parent as that's the one we build the features around and replace the pedestal one as a subtype of that.

**Comment by SpironoZeppeli** [comment]
I discussed this issue with Naksu on the discord. When you use a wrench on a crystal, nothing happens now. He said he refers it because it removes a newbie trap and as such should be kept in. I have to agree.

**Comment by JJRcop** [comment]
I don't think this works at all

**Comment by PKPenguin321** [comment]
You put your name after the first :cl: not a title

File: code/modules/power/supermatter/supermatter.dm (hunk 1 above)
[the remark this answers is not in the document]
Comment (optimumtact) [reply code/modules/power/supermatter/supermatter.dm]: the whole type path is upside down yeah

**Comment by SpironoZeppeli** [comment]
I tested it, do you need to explicitly state that now? Lmao

**Comment by 81Denton** [comment]
>I don't think this works at all

It takes about 20 1/2 seconds to check out this branch and test it for yourself 🤔 

reeeeeee

**Comment by Supermichael777** [comment]
>ITT people who think PRs are in game

**Comment by optimumtact** [comment]
I don't understand why this PR is attracting as many smoothbrained comments as it is today, but can you all make sure you are posting things that are relevant and or accurate please.

**Comment by JohnGinnane** [comment]
I know it's a little late but a message saying whether it's anchored or not, when examining, would've been handy

**Pull request merged**